lib/iex/lib/iex/evaluator.ex

```diff
@@ -89,14 +89,31 @@ defmodule IEx.Evaluator do
 
     result =
       with {:ok, tokens} <- :elixir.string_to_tokens(input, line, column, file, opts),
-           {:ok, adjusted_tokens} <- adjust_operator(tokens, line, column, file, opts, last_op),
+           {:ok, adjusted_tokens, adjusted_op} <-
+             adjust_operator(tokens, line, column, file, opts, last_op),
            {:ok, forms} <- :elixir.tokens_to_quoted(adjusted_tokens, file, opts) do
         last_op =
           case forms do
             {:=, _, [_, _]} -> :match
             _ -> :other
           end
 
+        forms =
+          if adjusted_op != nil do
+            quote do
+              if Process.get(:iex_error?) == true do
```

Comment:
Do we need to keep the Proces.get inside the quote? In general I'd agree with you but I think we will be fine here because IEx runs in a closed loop (Read-Eval-Print) and I would prefer to avoid changing the user forms as much as possible!

Reply:
Yeah, that makes sense 👍